lib/hex_web/email/sender.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HexWeb.Email.Sender do
+  require Logger
+
+  def deliver_now(email) do
+    ses_rate = Application.get_env(:hex_web, :ses_rate) |> String.to_integer
+
+    email
+    |> recipients
+    |> recipient_chunks(ses_rate)
+    |> Enum.each(fn recipients ->
+      HexWeb.Throttle.wait(HexWeb.SESThrottle, length(recipients))
+      HexWeb.Mailer.deliver_now(email)
```

Comment:
I was actually testing this and it seems like that's exactly what's happening. Adding a test is trivial, but actually pulling off the mail throttling not really. Actually, I am not sure if there's a way to achieve this without hacking Bamboo.

I would really appreciate if you have any suggestions, otherwise I'll have a dive in Bamboo's code and see what's the easiest approach.

/cc @ericmj @wojtekmach